lab/src/MyLinkedList.java

```diff
@@ -0,0 +1,94 @@
+import java.util.Iterator;
+
+public class MyLinkedList<T> implements Iterable<T>{
+
+	private Node<T> head;
+	private int size = 0;
+
+	public int getSize() {
+		return size;
+	}
+
+	public boolean remove(int index) {
+		if (index > size || index < 0) {
+			return false;
+		}
+		Node<T> nodeBefore = head;
+		Node<T> nodeAfter;
+		for ( int i = 0; i < index-1; i++) {
+			nodeBefore = nodeBefore.getNext();
+		}
+		nodeAfter = nodeBefore.getNext().getNext();
+		nodeBefore.setNext(nodeAfter);
```

Comment:
What if `index == 0`, i.e. if the caller wants to remove the head? I don't think your method will do that